core/templates/dev/head/pages/exploration-editor-page/improvements-tab/feedback-improvement-card/feedback-improvement-card.directive.ts

```diff
@@ -36,16 +36,18 @@ angular.module('oppia').directive('feedbackImprovementCard', [
             var numMessages = $scope.getData().messages.length;
             if (numMessages > 0) {
               var latestMessage = $scope.getData().messages[numMessages - 1];
+              var isNewlyOpenedThread =
```

Comment:
threadIsNewlyOpened (our convention is to start with a verb only for functions/methods)

Reply:
Done.